R/print.R

```diff
@@ -94,13 +100,12 @@ print.gsplot <- function(x, ...){
 #' @param \dots additional arguments (not used)
 #' @keywords internal
 print.view <- function(x, ...){
-  plots <- remove_field(x, param = c('window','grid'))
-  window <- x[['window']]
+  plots <- remove_field(x, param = c('par','grid','window'))
   
-  if(window$frame.plot){
-    box()
-  } 
-  par(window[['par']])
+  # if(window$frame.plot){
```

Comment:
Yes, and I wanted to bring up where we want to capture/store this. It is the boolean for box=TRUE but it isn't a par argument, it is a plot.window argument. It probably should be within view, but I punted on it for now